spiderable_server.js

```diff
@@ -4,6 +4,10 @@ var querystring = Npm.require('querystring');
 var urlParser = Npm.require('url');
 var crypto = Npm.require('crypto');
 
+var bound = Meteor.bindEnvironment(function(callback) {
```

Comment:
`Meteor.bindEnvironment` - is Directly uses Fibers
`Meteor.wrapAsync` - Works in a bit different way 
They both should work, but I believe `Meteor.bindEnvironment` more corrected to be used here.
It has a name `bound` because we're binding current vars state to the function context, but I'm open for discussion on it.